config/root/clusterworkspacetype-organization.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: tenancy.kcp.dev/v1alpha1
 kind: ClusterWorkspaceType
 metadata:
-  name: organization
+name: organization
 spec:
-  initializers:
-  - tenancy.kcp.dev/organization
+initializers:
+- tenancy.kcp.dev/organization
+defaultSubWorkspaceType: Team
```

Comment:
this changes the behaviour in the types installed upstream kcp by default. Intentional? Would leave Universal here.